website/i18n/en.json

```diff
@@ -0,0 +1,41 @@
+{
+  "_comment": "This file is auto-generated by write-translations.js",
+  "localized-strings": {
+    "next": "Next",
+    "previous": "Previous",
+    "tagline": "Cross Platform React Native UI Toolkit",
+    "avatar": "Avatar",
+    "badge": "Badge",
+    "button_group": "ButtonGroup",
+    "button": "Button",
+    "card": "Card",
+    "checkbox": "CheckBox",
+    "divider": "Divider",
+    "forms": "Forms",
+    "getting_started": "Getting Started",
+    "Getting Started": "Getting Started",
+    "header": "Header",
+    "icon": "Icon",
+    "input": "Input",
+    "lists": "Lists",
+    "overlay": "Overlay",
+    "overview": "Overview",
+    "pricing": "Pricing",
+    "rating": "Rating",
+    "searchbar": "SearchBar",
+    "slider": "Slider",
+    "social_icon": "SocialIcon",
+    "text": "Text",
+    "tile": "Tile",
+    "Docs": "Docs",
+    "Components": "Components",
+    "Help": "Help",
+    "Github": "Github",
+    "It Starts Basic": "It Starts Basic"
+  },
+  "pages-strings": {
+    "Help Translate|recruit community translators for your project": "Help Translate",
+    "Edit this Doc|recruitment message asking to edit the doc source": "Edit",
+    "Translate this Doc|recruitment message asking to translate the docs": "Translate"
+  }
+}
```

Comment:
So if we add new jsons it will automatically be translated ? What do we need to add if we want to add a new language for example ?

Reply:
You can take a look at the docs here https://docusaurus.io/docs/en/translation.html. I know it's possible because jest does it. https://facebook.github.io/jest/